2022-11-25 20:27:55,330 (841756/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2022-11-25 20:28:03,319 (841756/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/ej/.local/lib/python3.10/site-packages/pgcli/main.py", line 640, in connect\n    raise e\n  File "/home/ej/.local/lib/python3.10/site-packages/pgcli/main.py", line 627, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/ej/.local/lib/python3.10/site-packages/pgcli/pgexecute.py", line 161, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/ej/.local/lib/python3.10/site-packages/pgcli/pgexecute.py", line 207, in connect\n    conn = psycopg.connect(conn_info)\n  File "/home/ej/.local/lib/python3.10/site-packages/psycopg/connection.py", line 727, in connect\n    raise ex.with_traceback(None)\npsycopg.OperationalError: connection failed: FATAL:  database "ej" does not exist\n'
2022-11-25 20:28:12,168 (841885/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2022-11-25 20:28:16,832 (841885/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/ej/.local/lib/python3.10/site-packages/pgcli/main.py", line 640, in connect\n    raise e\n  File "/home/ej/.local/lib/python3.10/site-packages/pgcli/main.py", line 627, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/ej/.local/lib/python3.10/site-packages/pgcli/pgexecute.py", line 161, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/ej/.local/lib/python3.10/site-packages/pgcli/pgexecute.py", line 207, in connect\n    conn = psycopg.connect(conn_info)\n  File "/home/ej/.local/lib/python3.10/site-packages/psycopg/connection.py", line 727, in connect\n    raise ex.with_traceback(None)\npsycopg.OperationalError: connection failed: FATAL:  database "ej" does not exist\n'
2022-11-25 20:29:05,936 (842204/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2022-11-25 20:29:56,542 (842399/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2022-11-25 20:30:26,955 (842523/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2022-11-25 20:31:03,758 (842523/MainThread) pgcli.pgexecute ERROR - sql: 'create table master(', error: SyntaxError('syntax error at end of input\nLINE 1: create table master(\n                            ^')
2022-11-25 20:31:03,761 (842523/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/ej/.local/lib/python3.10/site-packages/pgcli/pgexecute.py", line 373, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/home/ej/.local/lib/python3.10/site-packages/pgcli/pgexecute.py", line 423, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/home/ej/.local/lib/python3.10/site-packages/pgcli/pgexecute.py", line 69, in execute\n    super().execute(*args, **kwargs)\n  File "/home/ej/.local/lib/python3.10/site-packages/psycopg/cursor.py", line 725, in execute\n    raise ex.with_traceback(None)\npsycopg.errors.SyntaxError: syntax error at end of input\nLINE 1: create table master(\n                            ^\n'
2022-11-25 20:31:28,601 (842523/MainThread) pgcli.pgexecute ERROR - sql: 'show tables', error: UndefinedObject('unrecognized configuration parameter "tables"')
2022-11-25 20:31:28,602 (842523/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/ej/.local/lib/python3.10/site-packages/pgcli/pgexecute.py", line 373, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/home/ej/.local/lib/python3.10/site-packages/pgcli/pgexecute.py", line 423, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/home/ej/.local/lib/python3.10/site-packages/pgcli/pgexecute.py", line 69, in execute\n    super().execute(*args, **kwargs)\n  File "/home/ej/.local/lib/python3.10/site-packages/psycopg/cursor.py", line 725, in execute\n    raise ex.with_traceback(None)\npsycopg.errors.UndefinedObject: unrecognized configuration parameter "tables"\n'
2022-12-03 16:41:21,330 (317202/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2022-12-03 17:19:58,448 (327095/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
